docs/guide/helper-html.md

```diff
@@ -92,21 +93,23 @@ if ($type === 'success') {
     Html::addCssClass($options, ['btn-success', 'btn-lg']);
 }
 
-echo Html::tag('div', 'Save', $options); // renders `class="btn btn-success btn-lg"`
+echo Html::tag('div', 'Save', $options);
+// renders '<div class="btn btn-success btn-lg">Save</div>'
 ```
 
-`Html::addCssClass()` prevents duplicating classes, so you don't need to worry the same class may appear twice:
+`Html::addCssClass()` prevents duplicating classes, so you don't need to worry if the same class may appear twice:
```

Comment:
'worry if ...' is almost equivalent to 'worry about the possibility that ...'.
So you don't need a comma before 'if'.